src/main/java/org/jabref/logic/importer/fileformat/RisImporter.java

```diff
@@ -216,13 +210,39 @@ public ParserResult importDatabase(BufferedReader reader) throws IOException {
                             comment = comment + " ";
                         }
                         comment = comment + value;
-                    }
-                    // Added ID import 2005.12.01, Morten Alver:
-                    else if ("ID".equals(tag)) {
+                    } else if ("ID".equals(tag)) {
                         fields.put("refid", value);
                     } else if ("M3".equals(tag) || "DO".equals(tag)) {
                         addDoi(fields, value);
                     }
+                    // fields for which there is no direct mapping in the bibtext standard
+                    else if ("AV".equals(tag)) {
+                        fields.put("archive_location", value);
+                    } else if ("C3".equals(tag)) {
+                        fields.put("event", value);
+                    } else if ("CN".equals(tag) || "VO".equals(tag)) {
+                        fields.put("call-number", value);
+                    } else if ("DB".equals(tag)) {
+                        fields.put("archive", value);
+                    } else if ("N1".equals(tag) || "RN".equals(tag)) {
+                        fields.put("note", value);
```

Comment:
Note is a standard biblatex field and thus should be added to `FieldName`

Reply:
fixed